build_tools/setup_helpers/extension.py

```diff
@@ -38,6 +38,7 @@ def _get_build(var, default=False):
 _BUILD_KALDI = False if platform.system() == 'Windows' else _get_build("BUILD_KALDI", True)
 _BUILD_TRANSDUCER = _get_build("BUILD_TRANSDUCER")
 _USE_ROCM = _get_build("USE_ROCM")
+_WITH_CUDA = True if torch.cuda.is_available() else False
```

Comment:
PyTorch core uses `USE_CUDA`. https://github.com/pytorch/pytorch/search?l=CMake&q=USE_CUDA 
So we should align with them.